### PR TITLE
Consistently use humantime in config files

### LINKED ISSUE #1821: Consistent use of durations in configs and chainspec
CC @Fraser999

### PR DESCRIPTION
**Pull request by rafal-ch**
* Use human-readable time units in all parameters that relate to duration in the `config.toml` file (`"30sec"`, `"120min"`, etc.)
* Code that handles these parameters no longer uses the raw integer data types, but the dedicated `TimeDiff` type

This PR introduces potentially breaking change - names of three parameters in config file have changed:
* `finished_entry_duration_secs` => `finished_entry_duration`
* `gossip_request_timeout_secs` => `gossip_request_timeout`
* `get_remainder_timeout_secs` => `get_remainder_timeout`

Closes #1821.

### CODE REVIEW

File: node/src/components/fetcher/tests.rs [hunk 1]
```diff
@@ -7,13 +7,14 @@ use casper_node_macros::reactor;
...
 use thiserror::Error;
-use tokio::time;
 
 use casper_types::ProtocolVersion;
 
 use super::*;
 use crate::{
-    components::{deploy_acceptor, in_memory_network::NetworkController, storage},
+    components::{
+        deploy_acceptor, in_memory_network::NetworkController, storage, tests_common::advance_time,
```
Comment (Fraser999) [inline node/src/components/fetcher/tests.rs]: Our convention is to not fully import functions, so this should be just `tests_common` and will become `tests_common::advance_time(time_to_advance).await;` at the callsite below.

File: node/src/components/gossiper/tests.rs [hunk 2]
```diff
@@ -25,6 +25,7 @@ use crate::{
         deploy_acceptor::{self, DeployAcceptor},
         in_memory_network::{self, InMemoryNetwork, NetworkController},
         storage::{self, Storage},
+        tests_common::advance_time,
```
Comment (Fraser999) [inline node/src/components/gossiper/tests.rs]: Same point here.

File: node/src/components/tests_common.rs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+use std::time::Duration;
```
Comment (Fraser999) [inline node/src/components/tests_common.rs]: We have a `testing` module in the root of node which is feature-gated behind `test` already.  I think this function should live there.

**Review by Fraser999 — COMMENTED** [review]
We have a `resources/production/config-example.toml` which should be updated in this PR too.

Other than the nitpick comments, this LGTM.

**Review by Fraser999 — APPROVED** [review]
LGTM

**Review by darthsiroftardis — APPROVED** [review]
LGTM!!

**Review by nilz3ro — APPROVED** [review]

**Pull request merged**